tools/gluc/sources/main.cpp

```diff
@@ -270,8 +365,7 @@ int main(int argc, char **argv)
                 continue;
             }
 
-            generateSystemImportPaths(argv[0]);
-            sema::constrainAST(ast, diagManager, ImportDirs);
+            sema::fastConstrainAST(ast, diagManager, &importManager);
```

Comment:
..................... fastConstrainAST doesn't constrain the whole AST....
add the needed functionality to sema::constrainAST, that's the main function for running Sema